nucliadb_utils/nucliadb_utils/audit/stream.py

```diff
@@ -229,6 +283,22 @@ async def search(
         auditrequest.trace_id = get_trace_id()
         await self.send(auditrequest)
 
+        self.kb_usage_utility.send_kb_usage(
+            service=Service.NUCLIA_DB,
+            account_id=None,
+            kb_id=kbid,
+            kb_source=KBSource.HOSTED,
+            # TODO unify AuditRequest client type and Nuclia Usage client type
+            searches=[
+                Search(
+                    client=ClientTypeKbUsage.Value(ClientType.Name(client_type)),  # type: ignore
+                    type=SearchType.SEARCH,
+                    tokens=2000,
```

Comment:
Out of curiosity: why 2000?
It could be a constant instead

Reply:
I didn't decide it, probably it will be adjusted later when they know which is the best number of tokens per search, and regarding a constant since it is only used here not sure if it is worth to add a constant, but as soon as this number is reused elsewhere then yes